docs/index.md

```diff
@@ -2,4 +2,74 @@
 title: Overview
 ---
 
-My new content here..
\ No newline at end of file
+This page is the top level entry point into the Substrate documentation.
+
+> **Some Expertise Needed**
+>
+>In order to get the most out of Substrate, you should have a good knowledge of blockchain concepts
+>and basic cryptography. Terminology like header, block, client, hash, transaction and signature
+>should be familiar. At present you will need a working knowledge of Rust to be able to do any
+>significant customization/adaption of Substrate (though eventually, we aim for this not to be the
+>case).
+
+Substrate is a blockchain development framework with a completely generic State Transition Function
+([STF](glossary.md#stf-state-transition-function)) and modular components for consensus, networking,
+and configuration.
+
+Despite being "completely generic", it comes with both standards and conventions - particularly with
+the Substrate Runtime Module Library ([SRML](conceptual/runtime/srml.md)) - regarding the underlying
+data-structures that power the STF, thereby making rapid blockchain development a reality.
+
+## Usage
+
+![Technical Freedom vs Development Ease](assets/technical-freedom.png)
+
+Substrate is designed to be used in one of three ways:
```

Comment:
Our users usually start with substrate node template to write their modules or add more srml modules, so that finally start a new chain. In this overview page, should we document such lifecycle? Here you are trying to tell user the three ways that they can play with substrate, but seems two of them (1, 3) are not very common usage.

Reply:
This is messaging coming from Gav. At this point in our documentation, we are not leading users to do a certain thing, just telling them what substrate can do. Creating modules with the SRML is a specific way to approach using Substrate.

Reply:
Fair point, the Substrate binary is only used in the ink! tutorial. However it uses all the modules and it is a good example. I think that it is correct to keep it high level. To me, this page is about showing Substrate features and not getting into details.